### ISSUE TITLE
Decide what to do about users with multiple MHV IDs.

### ISSUE THREAD

**Issue by patrickvinograd** [department-of-veterans-affairs/va.gov-team#19971]
# Problem
When a user authenticates and is issued an access token and we establish a mobile session for them, it's possible that IAM  may return us multiple MHV IDs. 

At that point, it's ambiguous which one we should use to invoke any MHV-based APIs such as secure messaging.

The MHV website solves the problem by preventing users from logging in and instructs them to contact the  help desk to resolve this situation. VA.gov recently implemented similar logic. 

Mobile is a little trickier because IAM  will happily issue the access and  refresh tokens for a user in this state, it's just that when we  go to use the access token to service an API  request, we may get  into  this situation.

# Possible Solutions
In order from most intrusive to least:

1. Get IAM to stop issuing access tokens at all for users in  this state. Would be consistent with MHV/VA.gov, but would block users from accessing any features even unrelated to MHV IDs. One upside is that there's a natural UX "place" for an error message to be  presented - namely in the auth flow.

2. Reject an API request if the introspected user session has duplicate MHV IDs. Would similarly be a logical match to VA.gov/MHV. But this means  any  unrelated API access,  such as for claims, would be blocked by having multiple MHV IDs that aren't even really relevant to claims. Also, this means that _any_ authenticated API request from the app needs to be capable of handling this scenario and  presenting something actionable to the user.

3. Reject only the API request that actually use the MHV ID. This is  probably the neatest, in the sense that  we can more easily explain whey a secure messaging  API request  failed due to an MHV ID issue. We still need to come up with a contract for the API to communicate back to the frontend the reason for the failure. Typically we use  specific  error codes to indicate issues like this.

**Comment by ayushchak** [comment]
How does VA.gov handle that today? Considered low priority atm.

**Comment by bridgethapner** [comment]
Rereading the ticket I see that VA.gov doesn't allow users in at all when this issue arises. I revisited a convo @patrickvinograd and I had in Slack in back in March to refresh my memory on this one & remembered that we discussed that this affects ~100-200 users per day currently. 

My POV is we need a solution for this soon. Questions I have include: 

1. What's the implication of taking the approach implemented in the app as of now? What frustrations or uncertainty could this cause for folks?
2. Can we find out a general overall percentage of users who are affected by this? 

I think understanding these things would help us determine priority.
